fury/actors/effect_manager.py

```diff
@@ -0,0 +1,431 @@
+import os
+import numpy as np
+from fury.actor import Actor, billboard
+from fury.colormap import create_colormap
+from fury.io import load_image
+from fury.lib import Texture, WindowToImageFilter
+from fury.shaders import (attribute_to_actor,
+                          compose_shader,
+                          import_fury_shader,
+                          shader_apply_effects,
+                          shader_custom_uniforms)
+from fury.ui import LineSlider2D, TextBlock2D, Panel2D
```

Comment:
Because this is the first PR for the effect manager, avoid adding any additional features to it. If you implement everything now, it could make creating tests and debugging your code more difficult.